### ISSUE TITLE
Question : how to modify collection variable using newman CLI

### ISSUE THREAD

**Issue by SureshRajpurohit85** [postmanlabs/newman#2968]
1. Newman Version (can be found via `newman -v`):
 2. OS details (type, version, and architecture):
 3. Are you using Newman as a library, or via the CLI?
 3. Did you encounter this recently, or has this bug always been there:
 4. Expected behaviour:
 5. Command / script used to run Newman:
 6. Sample collection, and auxiliary files (minus the sensitive details):
 7. Screenshots (if applicable):

**Comment by malvikach** [comment]
@SureshRajpurohit85 Do u want to write/update the collection JSON file. Please do share an example along with the Newman command that you are using to execute your collection.

**Comment by SureshRajpurohit85** [comment]
@malvikach  , during runtime , i want to modify the collection variable  using newman CLI command as its integral part of collection JSON, which i am not able to find option under newman. where as i can modify the env or global variable as they are part of different file

**Comment by malvikach** [comment]
@SureshRajpurohit85 I don't think it would be possible to edit the collection JSON during the runtime as of now. Could you show how you are editing the env or global variable. Maybe I can discuss the approach with the team to see some approaches for collection json too.

**Comment by victorlsds** [comment]
Is there any new status for this?
